packages/storage/src/indexed-db/index.ts

```diff
@@ -678,9 +682,12 @@ export class IndexedDb implements IndexedDbInterface {
     await tx.done;
   }
 
-  private determinePriceRelevanceThresholdForAsset(assetMetadata: Metadata): number {
+  private determinePriceRelevanceThresholdForAsset(
+    assetMetadata: Metadata,
+    epochDuration: bigint,
+  ): bigint {
     if (assetPatterns.delegationToken.capture(assetMetadata.display)) {
-      return PRICE_RELEVANCE_THRESHOLDS.delegationToken;
+      return epochDuration;
```

Comment:
Can we grab this from the `APP_PARAMETERS` table instead of passing it as an argument down?

Reply:
> Can we grab this from the `APP_PARAMETERS` table instead of passing it as an argument down?

We can, but that means we will read it from indexed-db for each asset , instead of reading it once

Reply:
Ah true